content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In the [model registry chapter](/doc/start/model-management/model-registry) we
+registered the model in the model registry and assigned it to some lifecycle
+stages. In this chapter, we will learn how to access and use models and how to
+use the model registry to trigger automated CICD model workflows.
+
+If you are using the example repository, the models are already versioned on a
+publicly readable DVC remote so you can access the model from there and use it.
+If you are instead using your own repository you need to set up your own DVC
+remote and push the data (including models) there. Have a look at our [Data
+management
+guide]((/doc/start/data-management/data-versioning#configuring-a-remote) to se
+how this is done.
+
+## Downloading models
+
+It is useful to download model artifacts for example for local testing or for
+use in CICD workflows. With models versioned by DVC this can be done easily by
+using the Studio UI.
+
+Go to the detailed view of your model, select the desired model version under
+the "Version info" and then click on the "Access model" button.
+
+Studio will present you with several ways of downloading models. Click on the
+"Download" tab to download the model directly from your browser. If you haven't
+done that yet, click on "Generate new token" to generate a DVC Studio token used
+to access the model on a DVC Remote. Finally, click on "Generate download links"
+and you will be able to download the model (the link will be valid for 1 hour).
+You can see all the steps here:
+
+![Download model](/img/mr-studio-download-model.gif)
+
+Alternatively, you can also download the model with the following CLI command:
+
+```bash
+dvc artifacts get https://github.com/<user>/example-get-started-experiments.git pool-segmentation
```

Comment:
Should we include configuring the studio token here?

Reply:
Yes and I guess we should also note that this is possible without Studio (with some extra work)